src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -711,14 +711,25 @@
                                 <RowDefinition Height="Auto" />
                             </Grid.RowDefinitions>
 
-                            <!--This Grid row contains the Package Paths section-->
+                            <!--This Grid row contains the Node and Package Paths section-->
                             <Expander x:Name="PackagePathsExpander"
                                         Grid.Row="0"
                                         Style="{StaticResource MenuExpanderStyle}" 
                                         IsExpanded="{Binding PreferencesTabs[Features].ExpanderActive, Converter={StaticResource ExpandersBindingConverter}, ConverterParameter=PackagePathsExpander}"
                                         Header="{x:Static p:Resources.PackagePathsExpanderName}">
                                 <StackPanel Orientation="Vertical" Margin="0,6,0,0">
-                                    <Label>SOME CONTENT</Label>
+                                    <Label  Content="{x:Static p:Resources.PreferencesViewPackageDownloadDirectory}" 
+                                            Padding="0,5,5,5"
+                                            FontSize="13"
+                                            Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                    <ComboBox Grid.Row="1" 
+                                              Width="Auto"
+                                              HorizontalAlignment="Left"
+                                              ItemsSource="{Binding Path=PackagePathsForInstall}"
+                                              SelectedItem="{Binding Path=SelectedPackagePathForInstall}"
+                                              Style="{StaticResource NoBordersComboBoxStyle}"
```

Comment:
what do you think about modifying this style so it has a dark background?
I added (or will add) this color to the modern dictionary:
https://github.com/DynamoDS/Dynamo/pull/11762/files#diff-094c8df98e6724d704bb015c259cdf5352917a2076c57a4b4ffccaa74d003c4dR2269

Reply:
I tried using this but there was no difference. Perhaps I'm using it incorrectly. I tried using it here by adding:
```
Background="{StaticResource PreferencesWindowItemDarkerBackgroundColor}"
```
and to the `NoBordersComboBoxStyle` in DynamoModern.xaml but neither worked. Any idea how I can use it for the combobox?

Reply:
Did you also add the new color? My PR has not been merged yet so it won't exist unless you also add it.

Reply:
Yes, I did add the color.

Reply:
I'm okay figuring this out later.